### DISCUSSION TITLE
Error when creating an android project

### DISCUSSION THREAD

**Discussion by VilmanVladimir** [MonoGame/MonoGame#8196] [Android]
Tell me what this error means and how to fix it.

NETSDK1181	Error getting pack version: Pack 'Microsoft.Android.Ref.31' was not present in workload manifests.	Project5	C:\Program Files\dotnet\sdk\8.0.200\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.FrameworkReferenceResolution.targets	90

**Comment by ThomasFOG** [comment]
You might want to check if you have the Android workload installed with VS2022 and/or the dotnet CLI with ```dotnet workload install android```.

**Reply by VilmanVladimir** [reply]
Thanks a lot
